data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3482 @@
+		dialog `"Here is your share : <payment>. You have saved my life, captain <first>, and I shall never forget it. You have earned the right to know my name : "Si-Ra". Come see me again soon if you are up for a challenge, I am sure that I will have what you want." Then he leaves, after doing what you assume to be a salute to you, his fist risen. You are taken by surprise by the act, and before you have the time to react, he is already leaving. He is probably in a hurry to buy that new ship he told you about.`
+
+
+
+mission "Unfettered: Attracted Attention"
+	invisible
+	landing
+	source "Freedom"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered: Prove your strength 2: active"
+	on offer
+		require "Jump Drive" 1
+	on enter "Almaaz"
+		dialog `The Unfettered comes to the bridge to warn you, "There are even more unfettered humans than anticipated. I told you not to come here with a jump drive!"`
+	npc evade
+		government "Pirate"
+		system "Almaaz"
+		personality disables plunders vindictive
+		fleet "Large Northern Pirates"
+
+mission "Unfettered: Jump Drive Retrieval"
+	invisible
+	landing
+	source "Freedom"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered: Prove Your Strength 2: active"
+	on offer
+		conversation
+			`Following the advice of the Hai, you park your ship close to the hidden depot, yet still close enough to the spaceport as to not draw unwanted attention by landing off grid. Your crew should be able to intervene should anything go wrong. Then you follow him to the hatch. He is moving at a quick pace, you have difficulties following him, even given the fact he is wearing what seems to be an heavy combat suit.`
+			choice
+				`	"Who made this vault, what should I expect?"`
+				`	"Should we not take more men to ensure our safety?"`
+					goto moremen
+				`	(No time for talking.)`
+					goto outsideship
+			`	"That is an information I cannot trust anyone with, not even you, and especially not here. I do not want the owner of this cache to be in danger whilst we need jump drives to expand. Assume it is owned by unfettered humans, or pirates as I think you call them, and that we are ready to face them," he tells you.`
+				goto outside
+			label moremen
+			`	"We are trying to be discrete here, not to draw attention! The more we are the more we will attract to attack us, we want them to think we have no value, not that we are rich fellows treasure hunting." Judging from his tone, this was a stupid question.`
+			
+			label outsideship
+			`	Before exiting, he warns you, "Whatever happens, do not try anything stupid, and stay behind me at all times." He seems worried and keeps looking around furtively, making as little sound as possible with his suit. He is impressively stealthy though, and you surmise the suit must have been crafted specifically for him in order to be so discrete. He does not have his helmet on as he peers cautiously around corners and assess his surroundings.`
```

Comment:
```suggestion
			label outsideship
			`	Before exiting, he warns you, "Whatever happens, do not try anything stupid, and stay behind me at all times." 
			`	Once outside he moves at a quick enough pace that you have difficulties following him. He seems worried and keeps looking around furtively, making as little sound as possible with his suit. He is impressively stealthy though, and you surmise the suit must have been crafted specifically for him in order to be so discrete. He depresses a switch every so often which pops out a mirror, allowing him to peer cautiously around corners and obstacles in your path.`
```
He would need to keep his helmet on unless he was willing to be identified as Hai on a human world. Also now you're outside of the ship you can refer to his speed.